cmd/juju-bridge/main.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/juju/juju/network/debinterfaces"
+	"github.com/juju/utils/clock"
+)
+
+const usage = `
+Bridge existing devices
+
+usage: [ -p ] [ -b <bridge-prefix ] <filename> <device-name>~<bridge-name>...
```

Comment:
Changed to '=' (why haven't I though of that?). It can't be ':' as it's often used for aliases.